parsons/action_network/action_network.py

```diff
@@ -595,14 +597,21 @@ def get_messages(self, limit=None, per_page=25, page=None, filter=None):
            filter:
                The OData query for filtering results. E.g. "modified_date gt '2014-03-25'".
                When None, no filter is applied.
+           unpack_statistics:
+                Whether to unpack the statistics dictionary into the table. Default to False.
 
 
         `Returns:`
-            A  JSON with all the messages related entries
+            A Parsons Table with all the messages related entries
         """
+        # Fetch messages
         if page:
             return self._get_page("messages", page, per_page, filter)
-        return self._get_entry_list("messages", limit, per_page, filter)
+        tbl = self._get_entry_list("messages", limit, per_page, filter)
+        # Unpack statistics
+        if unpack_statistics:
+            tbl.unpack_dict("statistics", prepend=False, include_original=True)
```

Comment:
I assume that based on the existing structure of the messages data Action Network returns, there's no chance of the unpacked statistics duplicating a column name? If there _is_ a chance the unpacked names would be duplicates, that might cause an error or just overwrite a column that the user might need. If you can reassure me I'll go ahead and merge this. :)

Reply:
That's correct. Here is the api documentation about what is returned. The statistics section has no overlapping field names with the other columns. 

[Action Network API Docs => Messages Endpoint](https://actionnetwork.org/docs/v2/messages)

Reply:
Great, thanks! I'll go ahead and merge.